### PR TITLE
update jetpack.io name in docs

### PR DESCRIPTION
**Pull request by Lagoja**
## Summary

Update jetpack.io in docs

## How was it tested?

Tested with `docusaurus build`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
 
-Devbox was originally developed by [jetpack.io](https://www.jetpack.io) and is internally powered by `nix`. 
+Devbox was originally developed by [Jetify](https://www.jetify.com) and is internally powered by `nix`. 
 
 ## Demo
 
 You can try out Devbox in your browser using the button below:
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/new)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/new)
```
Comment (LucilleH) [inline README.md:25]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/new)
```

File: docs/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 
 You can also test and contribute to our docs online using Devbox Cloud!
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetpack-io/devbox?folder=docs/app)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetify-com/devbox?folder=docs/app)
```
Comment (LucilleH) [inline docs/README.md:7]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetify-com/devbox?folder=docs/app)
```

File: docs/app/docs/devbox_cloud/index.mdx [hunk 3]
```diff
@@ -16,7 +16,7 @@ Devbox.sh provides a sandboxed playground environment for you to learn how Devbo
 
 You can launch a new playground from your browser by visiting [devbox.sh](https://devbox.sh), or click the button below to launch a quickstart shell:
 
-[![Open in Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/new)
+[![Open in Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/new)
```
Comment (LucilleH) [inline docs/app/docs/devbox_cloud/index.mdx:19]:
```suggestion
[![Open in Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/new)
```

File: docs/app/docs/devbox_cloud/index.mdx [hunk 4]
```diff
@@ -67,12 +67,12 @@ Click on the `Open in Desktop` button on the top right corner, which will open u
 
 If your project uses Devbox, you can make it easy for developers to test your project in a Devbox playground by adding the `Open in Devbox.sh` badge to your repo
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetpack-io/devbox)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetify-com/devbox)
```
Comment (LucilleH) [inline docs/app/docs/devbox_cloud/index.mdx:70]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetify-com/devbox)
```

File: docs/app/docs/devbox_cloud/index.mdx [hunk 5]
```diff
@@ -67,12 +67,12 @@ Click on the `Open in Desktop` button on the top right corner, which will open u
...
 
 Add the following code to your README.md:
 
 ```md
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/{path_to_repo})
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/{path_to_repo})
```
Comment (LucilleH) [inline docs/app/docs/devbox_cloud/index.mdx:75]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/{path_to_repo})
```

File: docs/app/docs/devbox_examples/databases/mariadb.md [hunk 6]
```diff
@@ -3,9 +3,9 @@ title: MariaDB
 ---
 MariaDB can be automatically configured for your dev environment by Devbox via the built-in MariaDB Plugin. This plugin will activate automatically when you install MariaDB using `devbox add mariadb`, or when you use a versioned Nix package like `devbox add mariadb_1010`
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/databases/mariadb)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/databases/mariadb)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/mariadb)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/mariadb)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/databases/mariadb.md:8]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/mariadb)
```

File: docs/app/docs/devbox_examples/databases/mysql.md [hunk 7]
```diff
@@ -3,9 +3,9 @@ title: MySQL
 ---
 MySQL can be automatically configured for your dev environment by Devbox via the built-in MySQL Plugin. This plugin will activate automatically when you install MySQL using `devbox add mysql80` or `devbox add mysql57`.
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/databases/mysql)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/databases/mysql)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetpack-io/devbox/?folder=examples/databases/mysql)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetify-com/devbox/?folder=examples/databases/mysql)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/databases/mysql.md:8]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/github.com/jetify-com/devbox/?folder=examples/databases/mysql)
```

File: docs/app/docs/devbox_examples/databases/postgres.md [hunk 8]
```diff
@@ -3,9 +3,9 @@ title: PostgreSQL
 ---
 PostgreSQL can be automatically configured by Devbox via the built-in Postgres Plugin. This plugin will activate automatically when you install Postgres using `devbox add postgresql`
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/databases/postgres)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/databases/postgres)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/postgres)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/postgres)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/databases/postgres.md:8]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/postgres)
```

File: docs/app/docs/devbox_examples/databases/redis.md [hunk 9]
```diff
@@ -4,9 +4,9 @@ title: Redis
 
 Redis can be configured automatically using Devbox's built in Redis plugin. This plugin will activate automatically when you install Redis using `devbox add redis`
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/databases/redis)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/databases/redis)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/redis)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/redis)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/databases/redis.md:9]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/redis)
```

File: docs/app/docs/devbox_examples/languages/csharp.md [hunk 10]
```diff
@@ -4,9 +4,9 @@ title: C# and .NET
 
 C# and .NET projects can be easily generated in Devbox by adding the dotnet SDK to your project. You can then create new projects using `dotnet new`
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/development/csharp)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/development/csharp)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/dotnet)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/dotnet)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/languages/csharp.md:9]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/dotnet)
```

File: docs/app/docs/devbox_examples/languages/elixir.md [hunk 11]
```diff
@@ -4,9 +4,9 @@ title: Elixir
 
 Elixir can be configured to install Hex and Rebar dependencies in a local directory. This will keep Elixir from trying to install in your immutable Nix Store:
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/development/elixir/elixir_hello)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/development/elixir/elixir_hello)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/elixir)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/elixir)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/languages/elixir.md:9]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/elixir)
```

File: docs/app/docs/devbox_examples/languages/fsharp.md [hunk 12]
```diff
@@ -4,9 +4,9 @@ title: F# and .NET
 
 F# and .NET projects can be easily generated in Devbox by adding the dotnet SDK to your project. You can then create new projects using `dotnet new`
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/development/fsharp)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/development/fsharp)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/fsharp)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/fsharp)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/languages/fsharp.md:9]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/fsharp)
```

File: docs/app/docs/devbox_examples/languages/go.md [hunk 13]
```diff
@@ -4,9 +4,9 @@ title: Go
 
 Go projects can be run in Devbox by adding the Go SDK to your project. If your project uses cgo or compiles against C libraries, you should also include them in your packages to ensure Go can compile successfully
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/development/go/hello-world)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/development/go/hello-world)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/go)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/go)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/languages/go.md:9]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/go)
```

File: docs/app/docs/devbox_examples/languages/java.md [hunk 14]
```diff
@@ -26,9 +26,9 @@ Other distributions of the JDK (such as OracleJDK and Eclipse Temurin) are avail
 
 ## Gradle
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/development/java/gradle/hello-world)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/development/java/gradle/hello-world)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/java-gradle)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/java-gradle)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/languages/java.md:31]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/java-gradle)
```

File: docs/app/docs/devbox_examples/languages/java.md [hunk 15]
```diff
@@ -79,9 +79,9 @@ An example `devbox.json` would look like the following:
 
 ## Maven
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/development/java/maven/hello-world)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/development/java/maven/hello-world)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/java-maven)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/java-maven)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/languages/java.md:84]:
```suggestion
[![Open In Devbox.sh](https://www.jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/java-maven)
```

File: docs/app/docs/devbox_examples/languages/nodejs.md [hunk 16]
```diff
@@ -39,15 +39,15 @@ To disable Corepack, remove the `DEVBOX_COREPACK_ENABLED` variable from your dev
 
 ### Yarn
 
-[**Example Repo**](https://github.com/jetpack-io/devbox/tree/main/examples/development/nodejs/nodejs-yarn)
+[**Example Repo**](https://github.com/jetify-com/devbox/tree/main/examples/development/nodejs/nodejs-yarn)
 
-[![Open In Devbox.sh](https://jetpack.io/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/node-yarn)
+[![Open In Devbox.sh](https://jetify.com/img/devbox/open-in-devbox.svg)](https://devbox.sh/open/templates/node-yarn)
```
Comment (LucilleH) [inline docs/app/docs/devbox_examples/languages/nodejs.md:44]: ditto ditto. canonical is `www`

File: docs/app/docs/quickstart.mdx [hunk 17]
```diff
@@ -117,22 +117,22 @@ We'll create a new development environment with the packages we need. These pack
...
-[![Built with Devbox](https://jetpack.io/img/devbox/shield_galaxy.svg)](https://jetpack.io/devbox/docs/contributor-quickstart/)
+[![Built with Devbox](https://jetify.com/img/devbox/shield_galaxy.svg)](https://jetify.com/devbox/docs/contributor-quickstart/)
 ```
 
 </TabItem>
 <TabItem value='html' label='HTML'>
 
 ```html
-<a href="https://jetpack.io/devbox/docs/contributor-quickstart/">
+<a href="https://jetify.com/devbox/docs/contributor-quickstart/">
```
Comment (LucilleH) [inline docs/app/docs/quickstart.mdx:133]: ditto

File: docs/app/docusaurus.config.js [hunk 18]
```diff
@@ -91,26 +91,26 @@ const config = {
         },
         footer: {
             links: [{
-                    title: "Jetpack.io",
+                    title: "Jetify",
                     items: [{
-                            label: "Jetpack",
-                            href: "http://jetpack.io"
+                            label: "Jetify",
+                            href: "http://jetify.com"
```
Comment (LucilleH) [inline docs/app/docusaurus.config.js:97]:
```suggestion
                            href: "http://www.jetify.com"
```

File: docs/app/docusaurus.config.js [hunk 19]
```diff
@@ -91,26 +91,26 @@ const config = {
...
                         },
                         {
                             label: "Blog",
-                            href: "http://jetpack.io/blog"
+                            href: "http://jetify.com/blog"
```
Comment (LucilleH) [inline docs/app/docusaurus.config.js:101]:
```suggestion
                            href: "http://www.jetify.com/blog"
```

File: vscode-extension/README.md [hunk 20]
```diff
@@ -1,6 +1,6 @@
 # devbox VSCode Extension
 
-This is the official VSCode extension for [devbox](https://github.com/jetpack-io/devbox) open source project by [jetpack.io](https://jetpack.io)
+This is the official VSCode extension for [devbox](https://github.com/jetify-com/devbox) open source project by [jetify.com](https://jetify.com)
```
Comment (LucilleH) [inline vscode-extension/README.md:3]:
```suggestion
This is the official VSCode extension for [devbox](https://github.com/jetify-com/devbox) open source project by [jetify.com](https://www.jetify.com)
```

**Review by LucilleH — APPROVED** [review]

File: docs/app/docs/contributor-quickstart.md [hunk 21]
```diff
@@ -72,12 +72,12 @@ Devbox requires the [Nix Package Manager](https://nixos.org/download.html). If N
...
 
 ### Use Devbox with your IDE
 * **[Direnv Integration](ide_configuration/direnv.md):** Devbox can integrate with [direnv](https://direnv.net/) to automatically activate your shell and packages when you navigate to your project.
-* **[Devbox for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=jetpack-io.devbox):** Install our VS Code extension to speed up common Devbox workflows or to use Devbox in a devcontainer.
+* **[Devbox for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=jetify-com.devbox):** Install our VS Code extension to speed up common Devbox workflows or to use Devbox in a devcontainer.
 
 ### Get Involved
-* **[Join our Discord Community](https://discord.gg/jetpack-io):** Chat with the development team and our growing community of Devbox users.
-* **[Visit us on Github](https://github.com/jetpack-io/devbox):** File issues and provide feedback, or even open a PR to contribute to Devbox or our Docs.
+* **[Join our Discord Community](https://discord.gg/jetify):** Chat with the development team and our growing community of Devbox users.
```
[the remark this answers is not in the document]
Comment (Lagoja) [reply docs/app/docs/contributor-quickstart.md:82]: That's what I'm planning on changing it to

**Pull request merged**